subscription/controls.py

```diff
@@ -364,6 +374,9 @@ def chargeForCyVerse(stripe_api_key, stripe_token, priceToCharge, stripeDescript
                 PaymentControl.sendCyVerseSyncFailedEmail(termLabel, username, purchaseDate, transactionId, error)
 
             PaymentControl.sendCyVerseEmail(msg, purchaseId, termLabel, partnerObj, emailAddress, firstname, lastname, priceToCharge, institute, transactionId, expirationDate, cardLast4, other)
+        else:
+            #send email to tech team for an error in purchase (possible duplicate)
+            PaymentControl.sendCyVersePaymentFailedEmail(username, charge_amount, message)
 
         if 'message' in message:
             PaymentControl.logPaymentError(partyId, username, emailAddress, message['message'])
```

Comment:
Need to log errMsg since message['message'] does not have a value for the duplicate user case